tools/castle-editor/code/framedesign.pas

```diff
@@ -2211,6 +2324,7 @@ procedure TDesignFrame.MenuItemViewportCameraSetInitialClick(Sender: TObject);
   V.AutoCamera := false;
 
   ModifiedOutsideObjectInspector;
+  RecordUndo('MenuItemViewportCameraSetInitialClick'); //whatever that means?
```

Comment:
Should be `RecordUndo('Camera Initial := Current');`

Reply:
Thanks, fixed!